pom.xml

```diff
@@ -105,6 +70,32 @@ since this is impossible for AWS S3 dependencies without Maven.
             </resource>
         </resources>
         <plugins>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>install</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>WEB-INF\lib</outputDirectory>
```

Comment:
Suggest keeping the path separator syntax consistent `WEB-INF/lib`. Here and on L93.